csp/middleware.py

```diff
@@ -1,10 +1,19 @@
 from django.conf import settings
 from django.utils.six.moves import http_client
+try:
```

Comment:
Nitpick, can we have an empty line before the try to separate this a bit better?